spec/koans.js

```diff
@@ -4,31 +4,39 @@ describe('`let` restricts the scope of the variable to the current block - ', ()
   describe('`let` vs. `var`.', () => {
     it('`var` works as usual, it does not restricts scope', () => {
       if (true) { 
-        /*You should add your code in here*/
+        var varX = true;
       }
-      //expect(varX).toBe(true);
+      expect(varX).toBe(true);
     }); 
 
     it('`let` restricts scope to inside the block', () => {
-      /*var or const? letX = false*/
+      const letX = false;
+      
       if (true) { 
-        /*var or const? letX = true*/
+        const letX = true
       }
-      //expect(letX).toBe(false);
+      expect(letX).toBe(false);
```

Comment:
Here to best demonstrate the ```let``` block scope it would be better like this: 
```
var letX = false;
if (true) { 
    let letX = true; 
}
```